docs/book/src/clusterctl/commands/adopt.md

```diff
@@ -1 +1,15 @@
 # clusterctl adopt
+
+## Pre-requisites
+
+In order for `clusterctl adopt` to work, ensure the components are correctly
+labeled as follows.
```

Comment:
labeled as follows. --> labeled; as a reference, you can consider the labels applied to the following providers.

Those are examples, other providers like e.g. CAPZ should use their own label